.gitignore

```diff
@@ -8,6 +8,9 @@ node_modules
 .npm
 .released-packages
 
+# Integration tests dependencies cache
```

Comment:
can we just merge this to master in another small PR or commit, else every commit to master will have to add this

Reply:
Merged in #304